doc/usermanual/overview/overview.xml

```diff
@@ -85,10 +85,9 @@
     </para></listitem>
 
     <listitem><para>
-      The powerful export system supports Google+ webalbum, flickr upload, disk storage, 1:1
-      copy, email attachments and can generate a simple html-based web gallery. darktable allows
-      you to export to low dynamic range (JPEG, JPEG2000, PNG, TIFF, PDF), 16-bit (PPM, TIFF),
-      or linear high dynamic range (PFM, EXR) images.
+      darktable can export developed images as low or high dynamic range files (JPEG, PNG, TIFF, 
+      PDF, PFM, EXR) to local disk storage, some web albums, LaTeX book template,
+      email attachments and can generate a simple html-based web gallery.
```

Comment:
i would add a comma between attachments and and

Reply:
comma was added